db/migrate/20160418141210_add_read_only_to_miq_alert.rb

```diff
@@ -1,9 +1,17 @@
+require 'miq_expression.rb'
+
 class AddReadOnlyToMiqAlert < ActiveRecord::Migration[5.0]
+  class MiqAlert < ActiveRecord::Base; end
+
+  def self.guids_from_yaml
+    YAML.load_file("db/fixtures/miq_alerts.yml").map { |h| h["guid"] || h[:guid] }
```

Comment:
I know this is the way it was, but I think this is wrong and makes it overly complicated.

* This migration shouldn't be using a db/fixtures file as it currently is because that file can change, so this migration could break if we change the way we seed alerts via that yml.  Instead, we should have a set of guids at the time of the migration (or now, it doesn't matter).
* seeding of alerts should be fixed to set read_only on [any new rows we create](https://github.com/ManageIQ/manageiq/blob/12e65ff2feb86382f5c894bf04fbd8a6572e251a/app/models/miq_alert.rb#L745) or we should add read_only to the yml file so any new rows created from our fixtures are marked as read_only.
* We won't need miq_expression if we do ☝️